superset/views/datasource.py

```diff
@@ -100,8 +98,8 @@ def external_metadata(self, datasource_type=None, datasource_id=None):
             database = (
                 db.session.query(Database).filter_by(id=request.args.get("db_id")).one()
             )
-            Table = ConnectorRegistry.sources["table"]
-            datasource = Table(
+            table = ConnectorRegistry.sources["table"]
```

Comment:
mmmh, this is actually a pointer to a class name, I'd either stick with `Table` or call it `table_cls` or equivalent

Reply:
I'm actually starting to use `table_cls` style, so +1 on that